cli/cmd/services.go

```diff
@@ -0,0 +1,417 @@
+			if status.Health == nil {
+				status.Health = &Component{
+					Name:    "server",
+					Status:  hr.Status,
+					Message: hr.Summary,
+				}
+			}
+		}
+	} else if serviceName == "rag" {
+		// For RAG, check via server's health endpoint
+		if hr, err := checkServerHealth(serverURL); err == nil {
+			if ragComp := findRAGComponent(hr); ragComp != nil {
+				status.Health = ragComp
+			}
+		}
+	}
+
+	return status
+}
+
+// getContainerDetails retrieves detailed information about a container
+func getContainerDetails(containerName string) (containerID, image, uptime string, err error) {
+	ctx := context.Background()
+	cli, err := createDockerClient()
+	if err != nil {
+		return "", "", "", fmt.Errorf("failed to create Docker client: %v", err)
+	}
+	defer cli.Close()
+
+	// Find the container by name
+	containers, err := cli.ContainerList(ctx, container.ListOptions{All: true})
+	if err != nil {
+		return "", "", "", fmt.Errorf("failed to list containers: %v", err)
+	}
+
+	var foundContainerID string
+	for _, c := range containers {
+		for _, name := range c.Names {
+			cleanName := strings.TrimPrefix(name, "/")
+			if cleanName == containerName {
+				foundContainerID = c.ID
+				break
+			}
+		}
+		if foundContainerID != "" {
+			break
+		}
+	}
+
+	if foundContainerID == "" {
+		return "", "", "", fmt.Errorf("container %s not found", containerName)
+	}
+
+	// Inspect the container for detailed information
+	containerJSON, err := cli.ContainerInspect(ctx, foundContainerID)
+	if err != nil {
+		return "", "", "", fmt.Errorf("failed to inspect container: %v", err)
+	}
+
+	// Extract short container ID (first 12 characters)
+	shortID := foundContainerID
+	if len(shortID) > 12 {
+		shortID = shortID[:12]
+	}
+
+	// Extract image name
+	imageName := containerJSON.Config.Image
+
+	// Calculate uptime if container is running
+	uptimeStr := ""
+	if containerJSON.State.Running {
+		startTime, err := time.Parse(time.RFC3339Nano, containerJSON.State.StartedAt)
+		if err == nil {
+			duration := time.Since(startTime)
+			uptimeStr = formatDuration(duration)
+		}
+	}
+
+	return shortID, imageName, uptimeStr, nil
+}
+
+// formatDuration formats a duration in a human-readable format
+func formatDuration(d time.Duration) string {
+	if d < time.Minute {
+		return fmt.Sprintf("%ds", int(d.Seconds()))
+	} else if d < time.Hour {
+		return fmt.Sprintf("%dm", int(d.Minutes()))
+	} else if d < 24*time.Hour {
+		hours := int(d.Hours())
+		minutes := int(d.Minutes()) % 60
+		if minutes > 0 {
+			return fmt.Sprintf("%dh%dm", hours, minutes)
+		}
+		return fmt.Sprintf("%dh", hours)
+	} else {
+		days := int(d.Hours()) / 24
+		hours := int(d.Hours()) % 24
+		if hours > 0 {
+			return fmt.Sprintf("%dd%dh", days, hours)
+		}
+		return fmt.Sprintf("%dd", days)
+	}
+}
+
+// formatServicesStatus formats the status output in a human-readable format
+func formatServicesStatus(output *ServicesStatusOutput) {
+	fmt.Println()
```

Comment:
prefer using `OutputInfo()` or similar. if we ever use this in the TUI, this will break the viewport.